src/components/ebay-checkbox/README.md

```diff
@@ -0,0 +1,21 @@
+# ebay-checkbox
+
+## ebay-checkbox Usage
+
+```marko
+<ebay-checkbox/>
+```
+
+## ebay-checkbox Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`disabled` | Boolean | Yes |
```

Comment:
This would be `No` for stateful. We use "stateful" to mean that it can be updated through the `property-observer`.